apps/reader/views.py

```diff
@@ -1031,9 +1030,11 @@ def folder_rss_feed(request, user_id, secret_token, unread_filter, folder_slug):
         
     data = {}
     data['title'] = "%s from %s (%s sites)" % (folder_title, user.username, len(feed_ids))
-    data['link'] = "https://%s%s" % (
+    data['link'] = "%s://%s%s" % (
```

Comment:
So I can see why you're making this change, as the feed link should match the scheme. But how did you come up with this? Were you noticing this on the other end?